### PR TITLE
Ttl expiry

### LINKED ISSUE #329: [cleanup] Have proper `Duration` for TTL instead of `i64`

### PR DESCRIPTION
**Pull request by alexsnaps**
Fixes #329 
Also cleans up some `feature` stuff, tho tests rely on some async runtime, so that `cargo test --no-default-features` won't compile :(

### CODE REVIEW

File: limitador/src/counter.rs [hunk 1]
```diff
@@ -68,8 +68,8 @@ impl Counter {
         false
     }
 
-    pub fn seconds(&self) -> u64 {
-        self.limit.seconds()
+    pub fn window(&self) -> Duration {
```
Comment (didierofrivia) [inline limitador/src/counter.rs:72]: This is great, it was always too ambiguous to refer to `seconds`

**Comment by didierofrivia** [comment]
Looking good, still some conflicts with ` limitador/src/storage/distributed/mod.rs`

**Comment by alexsnaps** [comment]
> still some conflicts

Still...? ~Still~ Again some conflicts! 🤣

**Review by didierofrivia — APPROVED** [review]
🥦

**Pull request merged**